package.json

```diff
@@ -197,11 +197,15 @@
 		"packages/*/package.json": [
 			"wp-scripts lint-pkg-json"
 		],
+		"*.scss": [
+			"stylelint"
+		],
 		"*.js": [
 			"eslint"
 		],
 		"{docs/{root-manifest.json,tool/*.js},packages/{*/README.md,*/src/{actions,selectors}.js,components/src/*/README.md}}": [
-			"npm run docs:build"
+			"npm run docs:build",
+			"git add docs/manifest.json"
```

Comment:
That's not the only thing that can change with the `docs:build` command. There could be updates to other sections of the docs folder. I think it's better to abort committing if there's unstaged changes in that case.

Reply:
I'm not sure there is a way to abort the commit if there are unstaged changes. I think we'll just have to rely on the contributor to realize they need to stage and commit those things.

Reply:
I think we can pass a flag to `npm run docs:build` which would force the process to exit with an error in the case docs got updated. This would allow us to abort commit in a similar way it happens for `eslint`.